controlplane/kubeadm/api/v1alpha3/suite_test.go

```diff
@@ -58,8 +58,8 @@ var _ = BeforeSuite(func(done Done) {
 	By("bootstrapping test environment")
 	testEnv = &envtest.Environment{
 		CRDDirectoryPaths: []string{
-			filepath.Join("..", "..", "..", "..", "config", "bootstrap", "crd", "bases"),
-			filepath.Join("..", "..", "..", "..", "config", "controlplane", "crd", "bases"),
+			filepath.Join("..", "..", "config", "crd", "bases"),
+			filepath.Join("..", "..", "..", "..", "bootstrap", "kubeadm", "config", "crd", "bases"),
```

Comment:
```suggestion
			filepath.Join("../../../..", "bootstrap", "kubeadm", "config", "crd", "bases"),
```

Reply:
better to keep the original if Windows is in the plans for the distant future.

Reply:
+1 we use that style everywhere

Reply:
ok. Sounds good. 👍